### DISCUSSION TITLE
[Server 3.0.3] All shortened links are gone

### DISCUSSION THREAD

**Discussion by thepra** [shlinkio/discussions#67] [Help wanted] [answered]
I've setup the docker shlink server, following the documentation and got the API key through the command line and started using it through the official shlink client for some time but now all the links are gone, before that the API key was receiving a 401 at each request from the client, so I figured that it did require a refresh probably, so I got again the key and put it in the client with further successful calls, but now every previous link is gone.

How is it possible? Is there a way to recover?

**Comment by acelaya** [comment, marked as answer]
It seems that something external whipped-out your database, probably around the time you started to get 401s with your API key.

The only possible option is to restore a database backup.

**Reply by thepra** [reply]
Mmmm, wait, I think I didn't had any DB setup in parameters on which the server should save it's data, all was running inside docker, and when it went down, the data followed along. My bad, closing up.
